packages/actions-global/src/hooks/use-on-action-error.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Dispatch } from 'redux';
+import { useDispatch } from 'react-redux';
+import { handleActionError } from '../actions';
+
+export default function useOnActionError(error) {
+  const dispatch = useDispatch<
+    Dispatch<ReturnType<typeof handleActionError>>
+  >();
+  return React.useCallback(() => {
+    const action = handleActionError(error);
+    if (action) dispatch(action);
```

Comment:
```suggestion
    if (action) return dispatch(action);
```

We should probably return this